src/modules/app/actions/listen-to-updates.js

```diff
@@ -100,12 +100,12 @@ export function listenToUpdates(history) {
           } else {
             console.log('OrderCanceled:', log)
             // if this is the user's order, then add it to the transaction display
-            if (log.orderCreator === getState().loginAccount.address) {
+            if (log.sender === getState().loginAccount.address) {
```

Comment:
Is `orderCreator` no longer included with the log?

Reply:
changed to sender from what I see with the newest version of augur-node.